UI.py

```diff
@@ -208,6 +208,27 @@ def CreateIsMemberWorkingOptions():
     ]
 
     return member_options
+
+def CreateIsEveryoneHereEmbed(approved_list_names, approved_list_reasons, unapproved_list, isAdmin):
+    try:
+        embed = discord.Embed(
+            title = f'Is Everyone Here',
+            description = f'Members not in meeting channel',
```

Comment:
```suggestion
            description = f'Reports if all relevant full-time members are in the meeting channel',
```
Descriptions should be descriptive